Google.PowerShell/Storage/GcsObject.cs

```diff
@@ -245,15 +245,15 @@ protected override void ProcessRecord()
 
             if (ParameterSetName == ParameterSetNames.ContentsFromFolder)
             {
-                // user gives us the path to a folder, we will resolve the path and upload the content of that folder
+                // User gives us the path to a folder, we will resolve the path and upload the contents of that folder.
                 string resolvedFolderPath = GetFullPath(Folder);
                 if (string.IsNullOrWhiteSpace(resolvedFolderPath) || !Directory.Exists(resolvedFolderPath))
                 {
-                    throw new DirectoryNotFoundException(string.Format("Directory {0} cannot be found", resolvedFolderPath));
+                    throw new DirectoryNotFoundException($"Directory '{resolvedFolderPath}' cannot be found.");
                 }
 
-                // call helper method to upload the content of the directory, the relative directory name of this directory
-                // will be its name
+                // Call helper method to upload the contents of the directory.
```

Comment:
"Call helper method to upload the contents of the directory." just describes what the code does, not _why_. The second comment is useful, since it helps the next programmer. The first isn't, as the code is pretty self-descriptive. So I'd remove the one on line 255.